flink-connectors/flink-connector-files/src/test/java/org/apache/flink/connector/file/src/FileSourceTextLinesITCase.java

```diff
@@ -119,6 +120,7 @@ private void testBoundedTextFileSource(FailoverType failoverType) throws Excepti
 
         final StreamExecutionEnvironment env = StreamExecutionEnvironment.getExecutionEnvironment();
         env.setParallelism(PARALLELISM);
+        env.setRestartStrategy(RestartStrategies.fixedDelayRestart(1, 50));
```

Comment:
I would like to understand why this change is necessary. Interestingly the `FileSourceTextLinesITCase` passes when I run the whole suite on `master` but `FileSourceTextLinesITCase.testBoundedTextFileSourceWithTaskManagerFailover` fails when executed individually. I think this looks like a test problem which we should fix separately.

Reply:
I have created https://issues.apache.org/jira/browse/FLINK-21155 to track this problem.

Reply:
Sure, that should be fine I suppose. I probably only ran into the issue when I debugged things in the IDE; CI is likely unaffected then.